### DISCUSSION TITLE
Build failure on ubuntu 22.04.4 LTS (Jammy Jellyfish)

### DISCUSSION THREAD

**Discussion by Hendriksn** [Project-OSRM/osrm-backend#7006] [Q&A]
I'm trying to build OSRM from source on my Ubuntu running on Windows (WSL). I followed the instructions, but the build process was unsuccessful.

Steps to Reproduce:

Installed all dependencies as stated in the documentation.
Cloned the OSRM repository.
Ran command:
cmake .. -DCMAKE_BUILD_TYPE=Release
cmake --build .

Errors: 
[build_output.log](https://github.com/user-attachments/files/16388350/build_output.log)

It does not seem to me that the erros are the same as it is here: [#6851](https://github.com/Project-OSRM/osrm-backend/discussions/6851)

Environment:

Ubuntu Version: 22.04.4 LTS
CMake Version: 3.22.1
GCC Version: 11.4.0
Can anyone please help me identify and resolve the problem? If you need further information, please let me know.

Thank you!

**Comment by jcoupey** [comment]
GCC 11 is no longer supported in CI builds, can you try building with GCC 12? It is available in Ubuntu 22.04.
